Description: install kwinwrapper, not westonwrapper
 .
 simplelogin (0.0.2+kubuntu1-0ubuntu1~ppa3) vivid; urgency=medium
 .
   * install kwinwrapper, not westonwrapper
Author: Boudewijn Rempt <boud@kde.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- simplelogin-0.0.2+kubuntu1.orig/CMakeLists.txt
+++ simplelogin-0.0.2+kubuntu1/CMakeLists.txt
@@ -40,6 +40,6 @@ target_link_libraries(simplelogin Qt5::C
 
 install(TARGETS simplelogin RUNTIME DESTINATION /usr/bin)
 install(FILES upstart/simplelogin.conf DESTINATION /etc/init)
-install(FILES westonwrapper DESTINATION /usr/bin)
+install(FILES kwinwrapper DESTINATION /usr/bin)
 install(FILES sddm-autologin DESTINATION /etc/pam.d)
 
